### ISSUE TITLE
ARKs should be visible to users 

### ISSUE THREAD

**Issue by faithc** [pulibrary/pulfalight#336]
On the current site, ARKs are displayed on the HTML/single-page view. We would like to continue to display this information somewhere on the finding aid so users can save/refer to its permanent URL. Perhaps on the collection summary page? @sdellis curious whether you have any suggestions for its placement? Thanks.

**Comment by tpendragon** [comment]
@regineheberlein @faithc Where are the ARKs? I'm not seeing them in aspace exports atm.

**Comment by regineheberlein** [comment]
@tpendragon They're in eadid/@url, e.g.
`<eadid url="http://arks.princeton.edu/ark:/88435/z029p473t">C0841</eadid>`

**Comment by tpendragon** [comment]
Thanks!
